e2e_tests/tests/cluster/test_rbac.py

```diff
@@ -59,18 +57,17 @@ def create_users_with_gloabl_roles(user_roles: List[List[str]]) -> List[authenti
     user_roles: list of roles to assign to each user, one entry per user.
     """
     user_creds: List[authentication.Credentials] = []
-    with logged_in_user(ADMIN_CREDENTIALS):
-        for roles in user_roles:
-            user = bindings.v1User(username=api_utils.get_random_string(), admin=False, active=True)
-            creds = api_utils.create_test_user(True, user=user)
-            for role in roles:
-                cli.rbac.assign_role(
-                    utils.CliArgsMock(
-                        username_to_assign=creds.username,
-                        role_name=role,
-                    )
-                )
-            user_creds.append(creds)
+    sess = api_utils.determined_test_session(admin=True)
+    for roles in user_roles:
+        user = bindings.v1User(username=api_utils.get_random_string(), admin=False, active=True)
+        creds = api_utils.create_test_user(True, user=user)
+        for role in roles:
+            api.assign_role(
```

Comment:
Excision of `determined.cli.rbac.assign_role` into `determined.common.api.assign_role` looks great, as does its use here. `api.assign_role` seems like a pretty reasonable place to call something from that fronts the backend. But there's something about it living in `determined.common.api._util` that feels wrong to me. It's a nit either way, and who really cares what _file_ the function is in, but if you think that it'll be helpful to us to someday have structure to that directory, I think now is a good time to start it.

Reply:
I really don't care what file the function is in.  If you want me to put it in a different file, lmk what to name it and I'll do it.

I do like having it in the `api` namespace, and this is a case where I really like the separation between how code is called and where code is defined.

Reply:
> I do like having it in the api namespace, and this is a case where I really like the separation between how code is called and where code is defined.

I've not had cause to differentiate the two in the past, but I agree. It's really nice, here.

This comment applies to many of the other moved functions, too. How about `determined/common/api/_rbac.py`?

Reply:
I think that sounds like a great name.

I put all the functions I added there, since they're all rbac-related.  Even not_found_errs is rbac-related.

Reply:
👍  I like it.